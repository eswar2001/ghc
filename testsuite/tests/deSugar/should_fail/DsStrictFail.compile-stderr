
DsStrictFail.hs:4:12: warning: [GHC-53633] [-Woverlapping-patterns (in -Wdefault)]
    Pattern match is redundant
    In a pattern binding: False = ...
