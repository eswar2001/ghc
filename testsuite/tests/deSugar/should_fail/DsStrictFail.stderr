DsStrictFail: DsStrictFail.hs:4:12-23: Non-exhaustive patterns in False

HasCallStack backtrace:
CallStack (from HasCallStack):
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Exception.hs:88:13 in ghc-internal:GHC.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Exception.hs:81:32 in ghc-internal:GHC.Exception
  throw, called at libraries/ghc-internal/src/Control/Exception/Base.hs:422:30 in ghc-internal:Control.Exception.Base



